src/main/webapp/app/exam/manage/suspicious-behavior/suspicious-sessions/suspicious-sessions.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { SuspiciousExamSessions, SuspiciousSessionReason } from 'app/entities/exam-session.model';
+import { StudentExam } from 'app/entities/student-exam.model';
+
+@Component({
+    selector: 'jhi-suspicious-sessions',
+    templateUrl: './suspicious-sessions.component.html',
+    styleUrls: ['./suspicious-sessions.component.scss'],
+})
+export class SuspiciousSessionsComponent implements OnInit {
+    @Input() suspiciousSessions: SuspiciousExamSessions;
+    suspiciousFingerprint = false;
+    suspiciousIpAddress = false;
+    suspiciousUserAgent = false;
+    ngOnInit(): void {
+        this.suspiciousFingerprint = this.suspiciousSessions.examSessions.some((session) => session.suspiciousReasons.includes(SuspiciousSessionReason.SAME_BROWSER_FINGERPRINT));
+        this.suspiciousIpAddress = this.suspiciousSessions.examSessions.some((session) => session.suspiciousReasons.includes(SuspiciousSessionReason.SAME_IP_ADDRESS));
+        this.suspiciousUserAgent = this.suspiciousSessions.examSessions.some((session) => session.suspiciousReasons.includes(SuspiciousSessionReason.SAME_USER_AGENT));
```

Comment:
Maybe put this `this.suspiciousSessions.examSessions.some(...)` into a helper function?